lighthouse-core/config/default.js

```diff
@@ -31,27 +40,6 @@ module.exports = {
       "http-redirect",
       "html-without-javascript"
     ]
-  }, {
-    "passName": "dbw",
-    "useThrottling": false,
-    "gatherers": [
-      "chrome-console-messages",
-      // "styles",
-      // "css-usage",
```

Comment:
ha, alright, it's more dangerous now though since uncommenting could really mess up everything in the perf pass